### ISSUE TITLE
conda-get-Path-prefix: Error: executing Command <path-to-env>

### ISSUE THREAD

**Issue by cakarmustactalone** [necaris/conda.el#104]
I have installed conda for my doom emacs but whenever I try to activate my environement I  could not handle this error :
`
conda--get-path-prefix: Error: executing command "/home/user/.conda/bin/conda ..activate "bash" "/home/user/.conda/envs/dataScience/"" produced error code 127`

How Can I solve this. That is really necessary for me bcause I use lsp servers on conda , not in system-wide!

Thanks, regards!

**Comment by cakarmustactalone** [comment]
I have tried these solution:
```
(custom-set-variables
 '(conda-anaconda-home "/opt/anaconda/"))

(setq conda-env-home-directory (expand-file-name "/home/jaguar/.conda/")
      conda-env-subdirectory "envs")

This time, I can switch to base environment but my other environment which is located at `~/.conda/envs/` is not shown in   `M-x conda-activate-env`

conda 4.10.1 btw.
```

**Comment by triccare** [comment]
Yes, it is conda 4.13.0.

Note: Though `conda` is in the path and, as mentioned any other `conda` command operates, the `conda activate` reports that the shell has not been initialized.  This is fixed by either forcing in the `-i` option along with whatever the value of `shell-comman-switch` is, or by changing the `command-format-string` to something like the following:
```
source ~/miniconda3/etc/profile.d/conda.sh && conda activate \"%s\"
```
This would be at this point in [conda.el](https://github.com/necaris/conda.el/blob/bb36ff70c56a9851cd5220700de51b2ac80d807e/conda.el#L214). The above drops the extra arguments to the format statement so only the environment is being passed. 

This seems to be connected to similar issues like #23, where conda has changed somehow to break things. I'm not exactly sure how to make the solution more general; the above patch seems to get the job done for me anyways. Hope it helps and if I get a chance to explore further, will try; the day-job is requiring attention.

**Comment by ltmsyvag** [comment]
I have a laziest fix ever:

```lisp
 (defun me/conda--get-path-prefix (env-dir)
  (expand-file-name "~/your_conda_folder/envs/your_env_name/bin"))

 (advice-add 'conda--get-path-prefix :override #'me/conda--get-path-prefix)
```

This works if you only stick to one conda convironment, as I do. My OS is Catalina.

**Comment by necaris** [comment]
Folks who have had this issue -- hopefully the updates in #23 will fix it. Please stay tuned!

**Comment by ltmsyvag** [comment]
> Folks who have had this issue -- hopefully the updates in #23 will fix it. Please stay tuned!

Great! I the meantime I guess the best "fix" is just `conda install conda=4.12`. Unless you have an unlikely critical dependency on the newest conda 4.13.
